react.js

```diff
@@ -27,21 +46,26 @@ this.state = {
   dislikes: ["mirrors", "garlic", "wooden stakes"]
 }
 
-
+const {name, home, dislikes} = this.state
 
 // 4. Write a React method that would add one and update the state of the count each time the method is called.
 
 this.state = {
   count: 0
 }
 
+countUp = () => {
+this.setState({ counter : this.state.counter +1 })
+}
+
 
 
 // 5. There are four mistakes in this code that would cause it to break our application. Find the mistakes and fix them:
 
 import React, { Component } from 'react';
+import App from './App.js'
 
-class Recipes{
+class Recipes extends Component{
   constructor(props){
     super(props)
     this.state = {
```

Comment:
In order to use map we have to have an array, so recipes should be an array of objects